src/components/loop-form.tsx

```diff
@@ -1,44 +1,52 @@
-import { makeStyles, TextField, Button } from '@material-ui/core';
+import { makeStyles, TextField, Typography, Button } from '@material-ui/core';
 import Autocomplete from '@material-ui/lab/Autocomplete';
 import React, { useState } from 'react';
 import { successMsg } from '../util/alerts';
+import fakePlaces from '../mock/data/places';
 
-interface robotLoopFormProps {
-  fleetName: string;
+interface LoopFormProps {
+  fleets: string[];
```

Comment:
What do you think about adding more descriptive names? it makes the difference when reading the code. E.g if I see without context that you're iterating fleets, I'll think that you're iterating an array of objects with the full content of the fleets and no an array of strings with the name of the fleets. I hope I made my point with this, but let me know if you have any questions.

what do you think about fleetNames, fleetDescriptions? 

I'm not good in assigning names either =)

Reply:
I understand! fleetNames would be a better option here since it's really just an array of the names, and not the full fleet object. will make the change!